obs-packaging/get_pkg_data.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#Copyright (c) 2018 Intel Corporation
+#
+#SPDX-License-Identifier: Apache-2.0
+#
+
+[ -z "${DEBUG}" ] || set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source ../scripts/lib.sh
+
+pkg=$1
+git_reference="1.1.0"
```

Comment:
hard-coded version? This name is a bit confusing - could you rename the variable to `qemu_version` or something?